test/check-application

```diff
@@ -76,7 +85,6 @@ class TestApplication(testlib.MachineCase):
 
         # Enable user service as well
         self.admin_s.execute("systemctl --user stop podman.service; systemctl --now --user enable podman.socket")
-        self.restore_dir("/home/admin/.local/share/containers", reboot_safe=True)
```

Comment:
Some form of this is still necessary. Does `podman system reset` do that? (I suppose it does)

Note that it's *not* the user dir which causes trouble, it's the system dir.